lib/components/user/places/favorite-place-page.js

```diff
@@ -52,7 +42,6 @@ class FavoritePlacePage extends Component {
    * Navigates back to the previous browser location,
    * whether it is the accounts page or another page that linked to the current URL.
    */
-  // TODO: Refine this behavior.
   _handleBack = () => {
     window.history.back()
   }
```

Comment:
Might be ok just to inline the `window.history.back()` now.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/280/commits/436b93da0ef77b9c75628f92d29cf8313085d163.